.travis/add_pod_annotations.sh

```diff
@@ -16,9 +16,9 @@ if $DIR/build-condition.sh $TRAVIS_COMMIT_RANGE projects/; then
     echo "Adding pod annotations"
 
     # Remove empty pod annotations
-    sed -i -e "s/podAnnotations: {}//" charts/hyperspace/values.yaml
+    sed -i -e "s/podAnnotations: {}//" charts/keycloak/values.yaml
```

Comment:
There is an error here (not introducted by this PR), because the empty pod annotation in values.yaml contains a space:
```suggestion
    sed -i -e "s/podAnnotations: { }//" charts/keycloak/values.yaml
```